Axi Back to Back example with good headers

khandelwaltanuj opened the pull request:
Hi @MikeOpenHWGroup 

There is a new PR, I have done a lot cleaning in this one. 
This contain an example of an AXI connected back to back. Master agent is connected to slave agent. 

I have removed many unsed files (19 now from 42 files to merge). I have added header everywhere except in do files. 
I have not move do files yet. 

thanks and regards
Tanuj Khandelwal

khandelwaltanuj commented on lib/cv_dv_utils/python/sim_cmd/compile.py:
```diff
@@ -0,0 +1,152 @@
+## ----------------------------------------------------------------------------
```
Hi @MikeOpenHWGroup 

This is new file, which did not exists before. 

Regards
Tanuj

MikeOpenHWGroup replied:
Then why does this file have a copyright header from Verilab?

khandelwaltanuj commented:
Hi @MikeOpenHWGroup 

For infomation: Each file comitted with this PR are new files which did not exists before in the github. 

Regards
tanuj khandelwal

khandelwaltanuj commented:
Hi @MikeOpenHWGroup 

I have removed now all references to varilab copyright. 

Regarsd
Tanuj

MikeOpenHWGroup reviewed (approved):
Thanks @khandelwaltanuj!

MikeOpenHWGroup commented:
Hi @ASintzoff, I am 99% confident that this pull-request will not impact CVA6 in any way because it is all new files that are not used in the CVA6 UVM environment.  However, I have been wrong before, so if you could also have a look that would be helpful.  Thanks.

ASintzoff reviewed (commented):
Is it possible to squash all "update header" commits to avoid such kind of commits in github.com/openhwgroup/core-v-verif repository?

khandelwaltanuj commented:
Hi @ASintzoff 

Thanks for the review. I have cleaned the README and removed the commencted lines. 
Though I have not idea on how to squash the multiple "header update" into one as they all concern different files and I made the changes using github editor. 

Thanks and regards
Tanuj Khandelwal

MikeOpenHWGroup commented:
> I have not idea on how to squash the multiple "header update" into one

This is a very useful 'git' skill!  Here are two ways to do it:
- [1](https://algocademy.com/blog/how-to-use-git-squash-for-streamlined-commit-history/) using the git command-line.
- [2](https://docs.github.com/en/desktop/managing-commits/squashing-commits-in-github-desktop) using the Github Desktop.

khandelwaltanuj commented:
Hi @MikeOpenHWGroup 

I have squashed the commits. 
Thanks and Regards
Tanuj Khandelwal

MikeOpenHWGroup commented:
Thanks @khandelwaltanuj.

Hi @ASintzoff, pls have another look.

ASintzoff commented:
I'm still puzzled by some commits (header updates). BTW, it will be better to not modify legal stuff if it can be the correct one in the first commit.

khandelwaltanuj commented:
Hi 

I have made another attempt at squashing some more header update commits.

Regards
Tanuj

khandelwaltanuj commented:
Hello,

Can we merge this one ? 

Regards
Tanuj

ASintzoff commented:
@khandelwaltanuj I would suggest you squash all commits as the first commit is adding some files which are removed in the second one.

For the rest, LGTM

khandelwaltanuj commented:
Hello @MikeOpenHWGroup 

I have done the squash required (I hope so this time this is correct) and I have added yaml based python script to compile a design. My attempt on a generic script to compile a design. Unfortunaly while doing a new push, I did not realise that the old pull request has not been merged. 

If its ok for you can we please merge this pull request ? 

Thanks and Regards
Tanuj
